ql/src/java/org/apache/hadoop/hive/ql/metadata/DummyPartition.java

```diff
@@ -83,4 +84,12 @@ public List<String> getValues() {
     return values;
   }
 
+  @Override
+  public boolean equals(Object obj) {
```

Comment:
why do you need this?

Reply:
👍

Reply:
is it ok to reuse parent’s hashcode? do we initialize tPartition?
````
public int hashCode() {
  return Objects.hashCode(tPartition);
}
````

Reply:
Added hashCode method in DummyPartition